src/index.html

```diff
@@ -0,0 +1,24 @@
+<!Doctype html>
+<html>
+  <head>
+    <link rel="stylesheet" href="./style.css"/>
+    <meta charset="text/html; utf-8"/>
+    <script src="calculator.js"></script>
+  </head>
+  <body>
+    <div class="calc-container">
+    <div class="calculator-wrapper">
+      <div class="calculator-display">
+        <div class="calculator-display-text" id="results">0</div>
+      </div>
+      <div class="calculator-buttons-wrapper" id='allButtons'>
+        <div class="calculator-button" id='clear'>C</div><div class="calculator-button" id='sign'>+/-</div><div class="calculator-button" id='percent'>%</div><div class="calculator-button calculator-button-operator" id='/'>÷</div>
+        <div class="calculator-button" id='seven'>7</div><div class="calculator-button" id='eight'>8</div><div class="calculator-button" id='nine'>9</div><div class="calculator-button calculator-button-operator" id='*'>x</div>
+        <div class="calculator-button" id='four'>4</div><div class="calculator-button" id='five'>5</div><div class="calculator-button" id='six'>6</div><div class="calculator-button calculator-button-operator" id='-'>—</div>
+        <div class="calculator-button" id="one">1</div><div class="calculator-button" id="two">2</div><div class="calculator-button" id='three'>3</div><div class="calculator-button calculator-button-operator" id='+'>+</div>
+        <div class="calculator-button calculator-wide-button" id='zero'>0</div><div class="calculator-button" id='dot'>.</div><div class="calculator-button calculator-button-operator calculator-bottom-right-button" id='equals'>=</div>
```

Comment:
I like the idea that the divs here are coded in the same way they are represented on the calculator, but it is also confusing. At first glance, I thought some ids were missing, but upon closer inspection, I noticed that there were several divs per line. I think one separate line per element would be clearer.